docs/cli.rst

```diff
@@ -0,0 +1,17 @@
+Casanovo CLI
+==============
+
+After installing the Casanovo package and all of its dependencies, the Casanovo CLI
```

Comment:
**suggestion:** For nicer git diffs, in text files you can split at the sentence level.